package.json

```
@@ -33,6 +33,7 @@
 		"test:jest": "jest --config='./tests/jest/jest.config.js'",
 		"scripts": "grunt scripts",
 		"scripts:watch": "grunt watch_scripts",
+		"scripts:packages:local": "export ELEMENTOR_PACKAGES_USE_LOCAL=1 && grunt scripts:packages:local",
```

```suggestion
		"scripts:packages:local": "cross-env ELEMENTOR_PACKAGES_USE_LOCAL=1 && grunt scripts:packages:local",
```

Maybe also add "npm run scripts:packages"

---

Done